Source/ConversationV1/Conversation.swift

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import Alamofire
+import Freddy
+
+public class Conversation {
+    
+    /// A WorkspaceID uniquely identifies a dialog application.
+    public typealias WorkspaceID = String
+    
+    private let username: String
+    private let password: String
+    
+    private let domain = "com.ibm.watson.developer-cloud.WatsonDeveloperCloud"
+    private let serviceURL = "https://gateway.watsonplatform.net/conversation/api"
+    
+    
+    public init(username: String, password: String) {
+        self.username = username
+        self.password = password
+    }
+    
+    private func dataToError(data: NSData) -> NSError? {
```

Comment:
This looks great. (Nothing to worry about now, but I suspect it will need to change in the future since the `{"code": ..., "error": ...}` model differs from the JSON error format of other services. That will probably be updated at some point with the Conversation service.)